src/components/icons/close-icon-mobile/close-icon-mobile.jsx

```diff
@@ -2,18 +2,18 @@ import React from "react";
 
 import PropTypes from "prop-types";
 
-export const CloseIcon = ({ className }) => (
+export const CloseIconMobile = ({ className }) => (
```

Comment:
Is possible to create a CloseIcon to be used in the MenuMobile and in the Newsletter that only changes a few props like className, width, etc?

Reply:
Yes, it is possible

Reply:
Done